src/components/views/settings/CrossSigningPanel.js

```diff
@@ -142,6 +157,12 @@ export default class CrossSigningPanel extends React.PureComponent {
                     {_t("Bootstrap cross-signing and secret storage")}
                 </AccessibleButton>
             </div>;
+        } else {
+            bootstrapButton = <div className="mx_CrossSigningPanel_buttonRow">
+                <AccessibleButton kind="danger" onClick={this._destroySecureSecretStorage}>
+                    {_t("Reset Secret Store and Cross-signing")}
```

Comment:
Let's use `secret storage` for consistency. Also, let's match the casing above, so perhaps:

```suggestion
                    {_t("Reset cross-signing and secret storage")}
```